src/credential/mod.rs

```diff
@@ -201,10 +421,39 @@ pub enum CredentialFormat {
     #[serde(rename = "jwt_vc_json-ld")]
     JwtVcJsonLd,
     LdpVc,
+    #[serde(rename = "vcdm2_sd_jwt")]
+    SdJwt,
```

Comment:
TODO: Do we still want to change this type to `VCDM2SdJwt` vs `SdJwt`. No objections either way.

Reply:
Yes please